packages/boba/subgraphs/twostep-withdrawal.json

```diff
@@ -0,0 +1,36 @@
+{
+  "version": "1",
+  "name": "twostep-withdrawal",
+  "abis": {
+    "L2StandardBridge": {
+      "path": "./abis/L2StandardBridge.json"
+    },
+    "L2ToL1MessagePasser": {
+      "path": "./abis/L2ToL1MessagePasser.json"
+    },
+    "OptimismPortal": {
+      "path": "./abis/OptimismPortal.json"
+    }
+  },
+  "chains": ["eth/bnb-boba-anchorage-l2", "eth/bnb-anchorage-l1"], // TODO
```

Comment:
comments in json might look unconventional, should we wait on this until the networks up?

Reply:
up to you :-) JSONs are invalid with comments anyway & subgraphs wouldn't be deployed as well until the network is live. Main reason to merge could be to avoid potential merge conflicts although here it's quite unlikely

Reply:
lets keep this open as a draft then to remind ourselves